README.md

```diff
@@ -1,3 +1,17 @@
 # crazy-bee-project
 
 ![Coverage](./frontend/coverage-badge.svg) ![Coverage](./backend/coverage-badge.svg)
+
+# Sample Hardhat Project
```

Comment:
We don't need to document a sample project in the main README doc. 
Please move these info into a new `.md` file inside the `/docs` folder.